agent/src/main/java/org/openjdk/jmc/agent/Agent.java

```diff
@@ -57,7 +57,7 @@
 	/**
 	 * This should be generated as part of the build later.
 	 */
-	public final static String VERSION = "0.0.2"; //$NON-NLS-1$
+	public final static String VERSION = "0.1.0"; //$NON-NLS-1$
```

Comment:
Perhaps we want to bump this to 0.9 or something, signifying that we're getting close to production ready?